artifactory/commands/npm/installorci_test.go

```diff
@@ -54,6 +56,43 @@ func TestPrepareConfigData(t *testing.T) {
 	}
 
 	// Assert that NPM_CONFIG__AUTH environment variable was set
-	assert.Equal(t, authToken, os.Getenv(npmConfigAuthEnv))
-	testsUtils.UnSetEnvAndAssert(t, npmConfigAuthEnv)
+	assert.Equal(t, authToken, os.Getenv(fmt.Sprintf(npmConfigAuthEnv, "//goodRegistry")))
+	testsUtils.UnSetEnvAndAssert(t, fmt.Sprintf(npmConfigAuthEnv, "//goodRegistry"))
+}
+
+func TestSetNpmConfigAuthEnv(t *testing.T) {
+	testCases := []struct {
+		name        string
+		com         *CommonArgs
+		value       string
+		expectedEnv string
+	}{
+		{
+			name: "set scoped registry auth env",
+			com: &CommonArgs{
+				npmVersion: version.NewVersion("9.5.0"),
```

Comment:
It is better to test the edge cases
```suggestion
				npmVersion: version.NewVersion("9.3.1"),
```

Reply:
done